#ifndef RELOAD_WEAPON_RESPONSE_INTERACTOR_H_
#define RELOAD_WEAPON_RESPONSE_INTERACTOR_H_

// game-models
#include "game-models/GameSession/game-session.h"
// response-models


namespace invasion::interactors {
	
class ReloadWeaponResponseInteractor {
	void execute() const;
};

} // namespace invasion::interactors



#endif // RELOAD_WEAPON_RESPONSE_INTERACTOR_H_
